CMakeLists.txt

```diff
@@ -1326,6 +1326,8 @@ if(WIN32)
   if(USE_WIN32_CRYPTO OR USE_SCHANNEL)
     list(APPEND CURL_LIBS "advapi32" "crypt32")
   endif()
+
+  list(APPEND CURL_LIBS "bcrypt")
```

Comment:
Same as my other 2 comments regarding `bcrypt` becoming an unconditional requirement to build curl.